packages/block-library/src/home-link/index.php

```diff
@@ -129,7 +129,7 @@ function block_core_home_link_build_li_wrapper_attributes( $context ) {
  */
 function render_block_core_home_link( $attributes, $content, $block ) {
 	if ( empty( $attributes['label'] ) ) {
-		return '';
+		$attributes['label'] = __( 'Home' );
```

Comment:
Let's maybe add a comment that this needs to be kept in sync with the `useEffect` thing in `edit.js` as I think we do elsewhere, too; and maybe another note to explain _why_ we're adding this fallback in the first place 🙂 

Something like

```suggestion
		// Using a fallback for the label attribute allows rendering the block even if no attributes have been set,
		// e.g. when using the block as a hooked block.
		// Note that the fallback value needs to be kept in sync with the one set in `edit.js` (upon first loading the block in the editor).
		$attributes['label'] = __( 'Home' );
```

(but feel free to tweak the verbiage).

Reply:
Yes, good point. I've accepted this suggestion.